workspaces/orchestrator/plugins/orchestrator-backend/src/service/OrchestratorService.ts

```diff
@@ -223,4 +223,17 @@ export class OrchestratorService {
     if (overview) overview.isAvailable = isWorkflowAvailable; // workflow overview is avaiable but the workflow itself is not
     return overview;
   }
+
+  public async pingWorkflowService(args: {
+    definitionId: string;
+  }): Promise<boolean | undefined> {
+    const { definitionId } = args;
+    const idUrlMap = await this.dataIndexService.fetchWorkflowServiceUrls();
```

Comment:
we shouldn't get the serviceUrl like this
it should request it only for current definitionId using 
https://github.com/redhat-developer/rhdh-plugins/blob/main/workspaces/orchestrator/plugins/orchestrator-backend/src/service/DataIndexService.ts#L132

Reply:
if fetchWorkflowInfo returns undefined, it should raise an error that the workflow doesn't exist